variables.md

```diff
@@ -325,6 +325,7 @@ PUBLIC_CLOUD_GCE_STACK_TYPE | string | IPV4_ONLY | Network stack type, possible
 PUBLIC_CLOUD_FIO | boolean | false | If set, storage_perf test module is added to the job.
 PUBLIC_CLOUD_FIO_RUNTIME | integer | 300 | Set the execution time for each FIO tests.
 PUBLIC_CLOUD_FIO_SSD_SIZE | string | "100G" | Set the additional disk size for the FIO tests.
+PUBLIC_CLOUD_CLOUD_INIT_FILE | string | "" | The name of `cloud-config.yaml` file located in `data/publiccloud/`
```

Comment:
I would say that variable can be just boolean . Anyway it is not possible to set any other value then hardcoded config file and introduction of another file will require another PR

Reply:
Yes, I get you. Can we speak about this on Monday? I'm not against it but I tried to keep this at least partially configurable. So `PUBLIC_CLOUD_CLOUD_INIT_FILE` should currently be `cloud-init.yaml` but we might quite likely need multiple files.

OR: We may have just one cloud-config yaml file as a `.eb` template. (I know this is used for dynamic autoyast xml files but I never did it)

Reply:
I'll change it.